src/olympia/scanners/migrations/0015_auto_20191112_1413.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 2.2.7 on 2019-11-12 14:13
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('scanners', '0014_alter_action_choices_on_scannerrule'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='scannerresult',
+            name='state',
+            field=models.PositiveSmallIntegerField(choices=[(0, 'Unknown'), (1, 'True positive'), (2, 'False positive')], default=0),
```

Comment:
Django doesn't add database defaults in column definitions (historical design decision, I believe because they want to keep control over the values in python). 

When deploying there will be a small window of time when we are running the old code from previous week tag on the new, updated database schemas from the newer tag. So, if you add a field that doesn't accept null values, the old code will fail when inserting new values as it doesn't know about the new column and it doesn't have a database default.

The moral of the story is: if we want proper backwards-compatibility without having to disable/re-enable scanners are deploy time, you want to make this field accept null values. 

You can either make this value one of the choices (something I've done in new fields for `AbuseReport`, using `null=True, blank=True, default=None`) or make it a separate temporarily accepted value you then migrate away from in a future deploy, but that's more extra work later.